### PR TITLE
Examine block'd armor readout

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

image

## Why It's Good For The Game

Way more readable

## Changelog

:cl: Melbert
qol: Block'd out armor readout, should be more readable now
/:cl:

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -390,12 +390,16 @@
 			if(flags_cover & PEPPERPROOF)
 				things_blocked += "pepperspray"
 			if(length(things_blocked))
-				readout += "\n<b>COVERAGE</b>"
-				readout += "\nIt will block [english_list(things_blocked)]."
+				readout += "<b><u>COVERAGE</u></b>"
+				readout += "It will block [english_list(things_blocked)]."
 
 		readout += "</span>"
```
Comment (san7890) [inline code/modules/clothing/clothing.dm:396]:
```suggestion
```

you removed the span class notice up there so we should also remove the span escape

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/clothing.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/clothing/clothing.dm:396]: Tru I even meant to and forgot

**Comment by carlarctg** [comment]
take the opportunity to replace the dumb roman numerals with percentages too

**Comment by MrMelbert** [comment]
soulless

**Comment by jughu** [comment]
can u show preview in lightmode

**Comment by alien3301** [comment]
> take the opportunity to replace the dumb roman numerals with percentages too

Someone doesn't know how to read roman numerals. Never been an issue for me.

**Comment by necromanceranne** [comment]
> take the opportunity to replace the dumb roman numerals with percentages too

I believe the roman numeral obsfucation is a deliberate headmin mandate.

**Comment by san7890** [comment]
white mode won't matter because this uses the default standard spans that everything else in the game already uses so there should not be a horrific difference

**Pull request merged**